core/txpool/legacypool/legacypool.go

```diff
@@ -246,26 +249,27 @@ type txpoolResetRequest struct {
 
 // New creates a new transaction pool to gather, sort and filter inbound
 // transactions from the network.
-func New(config Config, chain BlockChain) *LegacyPool {
+func New(config Config, chain BlockChain, auctioneerEnabled bool) *LegacyPool {
```

Comment:
put `auctioneerEnabled` in the mempool config instead of adding a parameter

Reply:
i do put it in the mempool config or do you mean add it in the chainConfig? cause that might be a good idea!